skaffold.yaml

```
@@ -4,7 +4,7 @@ build:
   local:
     useBuildkit: true
   artifacts:
-    - image: keptndev/helm-service
+    - image: keptn-contrib/helm-service
```

This might need to be
```suggestion
    - image: ghcr.io/keptn-contrib/helm-service
```